chef_master/source/chef_license_accept.rst

```diff
@@ -103,18 +115,31 @@ This tool walks users through the install and setup of Automate.
 The Chef license is accepted after that in the browser.
 Please follow the in-product prompts.
 
+Chef Infra Server
+-----------------------------------------------------
+When installed as a system package users accept the license with the ctl command, EG ``chef-server-ctl reconfigure --chef-license=accept``.
```

Comment:
What does EG mean?

Reply:
EG as in `for example` - Changed it to `E.G.`, does that help? Or would you prefer something else?

Reply:
`e.g.` is the proper notation. I use `e.g.` a lot personally, but I just realized that we're committed to using English rather than Latin in our documentation.

Reply:
Changed it for you! :)